src/Proto.Remote/Remote.cs

```diff
@@ -72,11 +72,9 @@ private static void SpawnEndpointManager(RemoteConfig config)
         {
             var props = Actor.FromProducer(() => new EndpointManager(config));
             EndpointManagerPid = Actor.Spawn(props);
+            EventStream.Instance.Subscribe<EndpointTerminatedEvent>(EndpointManagerPid.Tell);
         }

```

Comment:
In a real world scenario it is unlikely that unsubscribing is needed.
In a test scenario where we run many tests, it might be, maybe we should add a Remote.Shutdown() just to deal with this kind of scenarios (?)